cmd/podsync/config.go

```diff
@@ -50,6 +62,45 @@ type Log struct {
 	Compress bool `toml:"compress"`
 }
 
+// FetchConfig loads TOML configuration from a URL
+func FetchGistConfig(gistid string, token string) (*Config, error) {
```

Comment:
I'm a bit hesitant to add fetching configs from Gist.
Config typically contains API keys and Gists is a public place.
This will encourage ppl to publish their API keys on the web,
which is not good.

Reply:
agree. that's why I've added an option to read api keys from system envs

you can set `env:YOUTUBE_API_KEY` instead of api key string

Reply:
let's add a warning or error when someone uses gist and have api key published in it

Reply:
Ppl will publish gist first and then we'll see the warning :)
Possibly we could offer an example how to download gist via script and save it to a disk before running podsync?

Something like
```
curl gist > ~/config.toml
podsync --config ~/config.toml
```